### DISCUSSION TITLE
边上是否可以自定义图标?

### DISCUSSION THREAD

**Discussion by firstblood93** [antvis/X6#2629] [Q&A]
![image](https://user-images.githubusercontent.com/19219510/188408776-7303a1de-a508-4456-a0e8-4baa655f507a.png)

RT , 想在边上自定义图标 请问是否可以做到?官网案例没有找到符合条件的

**Comment by zmcode** [comment]
当然可以, 可以以label的形式渲染成图片, 自己创建一个makeup
https://x6.antv.vision/zh/examples/edge/label#render-with-html
或者可以注册成工具的形式
